Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/StartupHandler.java

```diff
@@ -36,10 +39,32 @@ public static void HandleInit(FragmentActivity parent)
     String[] gamesToLaunch = getGamesFromIntent(parent.getIntent());
     if (gamesToLaunch != null && gamesToLaunch.length > 0)
     {
+      // Open permission dialog, the game will be opened on the next attempt
+      if (!PermissionsHandler.hasWriteAccess(parent))
```

Comment:
`hasWriteAccess` always returns false under scoped storage. You need to support running Dolphin without permission to write to external storage being granted.